src/navigation/vertical-navigation-directive.js

```diff
@@ -43,8 +43,8 @@
  * @example
```

Comment:
The example was using JQuery to initialize the vertical navigation. Things like:
   $(document).ready(function()
I'm trying to use as little jQuery as possible throughout even in the examples.